config/nginx/nginx.conf

```diff
@@ -32,69 +37,52 @@ http {
   }
 
   server {
-    listen 443 ssl http2 default_server;
+    listen 443 default_server ssl;
 
-    ssl on;
     ssl_certificate     /etc/nginx/ssl/default-domain/fullchain.pem;
     ssl_certificate_key /etc/nginx/ssl/default-domain/privkey.pem;
-    ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols       TLSv1.2 TLSv1.3;
     ssl_prefer_server_ciphers on;
     ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
     ssl_dhparam /etc/nginx/ssl/dhparam.pem;
 
-    # Deny .* (e.g. .git/)
-    location ~ /\. {
-      deny all;
-    }
-
-    # Static files
     location / {
-      expires $expires;  #see "map" block above
-      root /etc/nginx/html;
-      try_files $uri $uri/index.html;
+        root /etc/nginx/html;
+        try_files $uri $uri/ =404;
     }
 
-    # Authentication
-    location /auth {
-      # Rate limit authentication
-      limit_req zone=login burst=20 nodelay;
+    location /api/ {
+        proxy_pass  http://api;
+        proxy_redirect   off;
+        proxy_http_version 1.1;
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto http;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+        proxy_connect_timeout 1m;
+        proxy_send_timeout 2m;
+        proxy_read_timeout 2m;
 
-      proxy_pass  http://auth/auth;
-      proxy_redirect   off;
-      proxy_http_version 1.1;
-      proxy_set_header Host $host;
-      proxy_set_header X-Real-IP $remote_addr;
-      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-      proxy_set_header X-Forwarded-Proto https;
-      proxy_set_header Upgrade $http_upgrade;
-      proxy_set_header Connection "upgrade";
-      proxy_connect_timeout 1m;
-      proxy_send_timeout 2m;
-      proxy_read_timeout 2m;
+        location /api/server/ {
+          proxy_pass  http://dispatch;
+        }
     }
 
-    location /api/ {
-      proxy_pass  http://api;
+    location /auth/ {
+      proxy_pass  http://auth;
       proxy_redirect   off;
       proxy_http_version 1.1;
-      proxy_set_header Host $host;
+      proxy_set_header Host $http_host;
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-      proxy_set_header X-Forwarded-Proto https;
```

Comment:
shouldn't this be HTTPS now? I think it was wrong previously
it's used as a flag to indicate if HTTPS is being used and has been terminated at nginx - this is now indicating that nginx is using HTTP which might actually cause more issues as it'll attempt to redirect to HTTPS